config/babel.test.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
```

Comment:
Instead of using the individual plugins for each syntax, I think it would make more sense to use the env preset and set it to our browser matrix. Take a look at: https://github.com/babel/babel/tree/master/packages/babel-preset-env

Reply:
I assume the motivation of this pr is that we want to get IE11 tests running first, but eventually we don't want to depend on babel for our code to work? I just feel it shows the gap more clearly this way. But I don't know, what do you think?

Reply:
I mean, for right now this is great. Using the preset-env plugin just gives us more confidence that we properly transpile **all** the syntaxes from our tests that would break some of our non-standard browsers.

Reply:
Your call.

Reply:
I agree. Done. Using @babel/preset-env that targets our browsers matrix now.

Store browsers in a standard .browserslistrc file now. Using babel 7 now, as only this new version knows to search for .browserslistrc file while doing karma preprocessing.